analytics/tests/metrics/test_burndown.py

```diff
@@ -356,7 +356,7 @@ def test_include_issues_closed_after_sprint_end(self):
         # validation
         assert output.stats.get(self.TOTAL_CLOSED).value == 2
         assert output.stats.get(self.TOTAL_OPENED).value == 3
-        assert output.stats.get(self.PCT_CLOSED).value == 66.67  # rounded to 2 places
+        assert output.stats.get(self.PCT_CLOSED).value == 66  # rounded to 2 places
```

Comment:
```suggestion
        assert output.stats.get(self.PCT_CLOSED).value == 67  # rounded to nearest whole number
```

Changed this sounds it reflects the rounding to the nearest whole number